semanticdb/scalac/library/src/main/scala/scala/meta/internal/semanticdb/scalac/TextDocumentOps.scala

```diff
@@ -672,6 +673,11 @@ trait TextDocumentOps { self: SemanticdbOps =>
                 if (gtree.pos != null && gtree.pos.isRange) {
                   tryNamedArg(gtree, gtree.pos.start, gtree.pos.point)
                 }
+                // fix #2040
+                gtree match {
+                  case view: g.ApplyImplicitView => traverse(view.args.head)
```

Comment:
Can we add a guard here just in case? My experience is that it's best to play it extra safe when working with compiler APIs
```suggestion
                  case view: g.ApplyImplicitView if view.args.nonEmpty => traverse(view.args.head)
```